SpecFlow.BDD.Selenium.POM.Automation.Mars/Automation.Mars.Core/WebElements/AtWebElement.cs

```diff
@@ -39,8 +39,8 @@ public IWebElement GetElement()
             }
```

Comment:
why was there a need to update this?

Reply:
Thanks for reviewing so thoroughly, Aman.
In this GetElement() function, I implemented explicit wait.
If the XPath of a webElement is right, it can make sure we get that element properly. Otherwise, the logger will print the error stack messages and the program will not suspend because of those exceptions like StaleElementReferenceException, NoSuchElementException, etc...
This function is as a basic function when operating a webElement like Click, SendKeys.